lms/static/sass/base/_reset.scss

```diff
@@ -96,4 +96,5 @@ td { vertical-align: top; }
   @page { margin: 0.5cm; }
   p, h2, h3 { orphans: 3; widows: 3; }
   h2, h3 { page-break-after: avoid; }
+  .donotprint { display:none !important; }
```

Comment:
1.) I'm in agreement with Sarina here that .donotprint should be removed prior to merge here. Authors can still use display: none if they would really like to exclude more additional special case content. 

2.) %ui-print-excluded { @media print { display: none} } 
I would add this to _extends.scss so that we can help make the styling easier to read and maintain. That way, any class you want to apply this to simply gets a @extend %ui-print-excluded rule. This also helps reduce cases where media queries are floating off far away from the class name they belong to (which is common in our old, very strangely nested css).